include/solvers/mpm_base.h

```diff
@@ -260,6 +269,12 @@ class MPMBase : public MPM {
   //! Locate particles
   bool locate_particles_{true};
 
+  //! Absorbing Boundary Variables
+  bool absorbing_boundary_{false};
+  std::vector<std::shared_ptr<mpm::AbsorbingConstraint>> absorbing_constraint_;
+  std::vector<int> absorbing_nset_id_;
```

Comment:
I am wondering whether the absorbing constraint can be put under the `constraints_` variable? I mean, is that necessary to create and store `absorbing_constraint_` and `absorbing_nset_id_`?

Reply:
vectors moved